pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -107,15 +106,8 @@ func ObserveClusterMembers(genericListers configobserver.Listers, recorder event
 
 			_, err := observer.listers.NodeLister.Get(nodeName)
 			if errors.IsNotFound(err) {
-				// if the node is no londer available we use the endpoint observatiopn
-				klog.Warningf("error: Node %s not found: adding remove status to %s ", nodeName, previousMember.Name)
-				clusterMember, err := setMember(previousMember.Name, previousMember.PeerURLS, ceoapi.MemberRemove)
-				if err != nil {
-					return existingConfig, append(errs, err)
-
-				}
-				observer.ObservedMembers = append(observer.ObservedMembers, clusterMember)
-				continue
+				klog.Warningf("error: Node %s not found", previousMember.Name)
+				return existingConfig, append(errs, err)
```

Comment:
what does this do and do we want to do it? Let's use an example of a node being terminated by mistake.

Reply:
We are returning the existingConfig. So, clustermember thinks nothing changed.

Reply:
We could just continue so that observedMembers be sent without including the terminated node.

Reply:
if a previous members pod was not found, we check the node if the node was not found we removed the member from cluster with MemberRemove. I think your idea makes sense.

Reply:
How about appending err to errs? 
```append(errs, err)```